lib/sqlalchemy/engine/interfaces.py

```diff
@@ -122,7 +122,7 @@ def close(self) -> None: ...
 
     def commit(self) -> None: ...
 
-    def cursor(self) -> DBAPICursor: ...
+    def cursor(self, *args: Any, **kwargs: Any) -> DBAPICursor: ...
```

Comment:
were the missing types causing some type issues?

Reply:
yes, on `mysql.base.create_server_side_cursor`, cursor() is called with parmeter `self.dialect._sscursor`.